scripts/db_migrator.py

```diff
@@ -51,6 +53,12 @@ def __init__(self, namespace, socket=None):
         self.TABLE_KEY       = 'DATABASE'
         self.TABLE_FIELD     = 'VERSION'
 
+        # load config data from minigraph to get the default/hardcoded values from minigraph.py
+        # this is to avoid duplicating the hardcoded these values in db_migrator
+        self.minigraph_data = None
+        if os.path.isfile(MINIGRAPH_FILE):
+            self.minigraph_data = parse_xml(MINIGRAPH_FILE)
```

Comment:
Do you need to protect against parse_xml throwing error?

Reply:
Sure. Added now.